### ISSUE TITLE
Pivot: underline not visible in High Contrast mode + firefox

### ISSUE THREAD

**Issue by AbhinavSinghT** [OfficeDev/office-ui-fabric-react#1944]
### Bug Report

- __Package version(s)__: 2.34.2
- __Browser and OS versions__: Windows 10 + FireFox

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: VSTS (CI/CD)

#### Describe the issue:

Many high contrast styles are not visible in FireFox, 
1. Underline does not appear for pivot.
Same issue got fixed for other browsers - https://github.com/OfficeDev/office-ui-fabric-react/issues/1183.
2. Selection of choice group not visible.
3. Check-box styles are different.

#### Actual behavior:
In High-Contrast + FireFox:
Pivot Tab does not have underline.
Choice group selection is not visible
Check box style is different

#### Expected behavior:

Underline should appear on selected pivot tab. Styles should be similar to edge/IE in high contrast.

### If applicable, please provide a codepen repro:

Open https://dev.office.com/fabric#/components/pivot in high contrast + FireFox.
